vue-frontend/src/components/partials/HeaderV2.vue

```diff
@@ -0,0 +1,231 @@
+        <router-link to="/" replace>
+          <div class="navbar-brand">
+            <img
+              :src="headerLogoImage"
+              class="header-logo-image mt-1"
+              alt="canopas-logo"
+            />
+          </div>
+        </router-link>
+        <div class="navbar-collapse">
+          <ul class="navbar-nav ms-auto mb-2 mb-lg-0">
+            <li class="nav-item-margin">
+              <a
+                class="nav-link v2-title-3-text"
+                href="/"
+                :style="[
+                  activeHomePath
+                    ? {
+                        textDecoration: `underline !important`,
+                        'text-underline-offset': '0.3rem',
+                      }
+                    : { textDecoration: `none` },
+                ]"
+                ><span :class="[activeHomePath ? '' : 'v2-underline-text']"
+                  >Ho</span
+                >me</a
+              >
+            </li>
+            <li class="nav-item-margin v2-title-3-text">
+              <a class="nav-link" :href="careerURL">Career</a>
+            </li>
+            <li class="nav-item-margin v2-title-3-text">
+              <a class="nav-link" :href="blogsURL" target="_blank">Blogs</a>
+            </li>
+            <li class="nav-item-margin v2-title-3-text">
+              <a class="nav-link" target="_blank">Portfolio</a>
+            </li>
+            <li>
+              <a
+                :href="contactURL"
+                class="nav-item-margin v2-title-3-text"
+                v-bind:class="activeContactPath ? 'v2-button' : 'v2-black-btn'"
+                >Let's Talk</a
+              >
+            </li>
+            <li>
+              <a
+                :href="contactURL"
+                class="nav-item-margin v2-title-3-text btn-link"
+                :style="[
+                  activeContactPath
+                    ? {
+                        textDecoration: `underline !important`,
+                        'text-underline-offset': '0.3rem',
+                      }
+                    : { textDecoration: `none` },
+                ]"
+                >Let's Talk
+              </a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+  </div>
+</template>
+
+<script type="module">
+import headerLogoImage from "@/assets/images/logo/logo-header.svg";
+import Config from "@/config.js";
+
+export default {
+  data() {
+    return {
+      headerLogoImage: headerLogoImage,
+      careerURL: "/jobs",
+      contactURL: "/contact",
+      blogsURL: Config.BLOG_URL,
+      navbarSticky: false,
+      navbarAnimation: false,
+      navContainerHeight: 133,
+      lastScrollY: 0,
+      activeHomePath: false,
+      activeCareerPath: false,
+      activeContactPath: false,
+    };
+  },
+  components: {},
+  mounted() {
+    let currentRoutePath = this.$router.currentRoute._value.path;
+
+    this.activeHomePath = currentRoutePath == "/";
```

Comment:
We can remove all separate variables if its taken in array and handle `active` on click